src/snowflake/snowpark/modin/plugin/docstrings/series_utils.py

```diff
@@ -7,13 +7,13 @@
 
 class StringMethods:
     def casefold():
-        pass
+        pass  # pragma: no cover
```

Comment:
Should we skip coverage for the entire classes instead, as [here](https://github.com/snowflakedb/snowpark-python/blob/903eee29597687ac64d4041c903c106d7a3e84a8/src/snowflake/snowpark/modin/plugin/docstrings/base.py#L388)?

Reply:
+1. This is what we do for all other object types, not just base. So better (and easier) to stay consistent.

Reply:
Yes, we skipped all in tox.ini. PTAL